### PR TITLE
feat(nova): add laravel nova docs extension

### PR DESCRIPTION
**Pull request by a-drew**
## Description

Adds a new extension to search through Laravel Nova documentation directly in raycast.

Relies on a mix of static sections seen for each version of nova on the site as well as searching against their algolia index.

## Screencast

Screen Shot 2022-04-08 at 11 22 31 AM

Screen Shot 2022-04-08 at 11 23 59 AM

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/laravel-nova/package.json [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  "description": "Laravel Nova Docs",
+  "icon": "command-icon.png",
+  "author": "plank",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "nova",
+      "title": "Search Documentation",
+      "subtitle": "Laravel Nova",
+      "description": "Search Laravel Nova docs directly from raycast.",
```
Comment (mattisssa) [inline extensions/laravel-nova/package.json:14]:
```suggestion
      "description": "Search Laravel Nova docs directly from Raycast.",
```

File: extensions/laravel-nova/src/nova.tsx [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+  OpenInBrowserAction,
+  showToast,
+  Toast,
+} from "@raycast/api";
+import { useEffect, useMemo, useState } from "react";
+import algoliasearch from "algoliasearch";
+import striptags from "striptags";
+
+const APPID = "BH4D9OD16A";
+const APIKEY = "5aa44fede3f10262000a8c4f046033d5";
```
Comment (mattisssa) [inline extensions/laravel-nova/src/nova.tsx:15]: is it alright to hardcode the APIKEY here? Won't there be any limitation for usage?

**Review by mattisssa — APPROVED** [review]

**Comment by mattisssa** [comment]
Hey @a-drew! I've moved your screenshot to the metadata folder so it could be used for the Store extension's screenshots. 
The only thing is that it has the wrong sizing, could you please update the size? Or just move it back to your `/media` folder if you don't want to use this image as the Store screenshot.

File: extensions/laravel-nova/src/nova.tsx (hunk 2 above)
Comment (a-drew) [reply extensions/laravel-nova/src/nova.tsx:15]: Not sure at all if I'm honest. I'm following the same pattern seen in other algolia based extensions / even in an Alfred workflow that I was using prior to Raycast.

https://github.com/raycast/extensions/blob/fd5ead772a1341cd2f2f6fcfa4e51eda7abcd9ab/extensions/laracasts/src/laracasts.tsx#L6-L8

https://github.com/raycast/extensions/blob/fd5ead772a1341cd2f2f6fcfa4e51eda7abcd9ab/extensions/laravel-docs/src/index.tsx#L39-L41

https://github.com/techouse/alfred-nova-docs/blob/master/bin/src/constants/config.dart#L7-L10

**Comment by mattisssa** [comment]
Cool, thank you!

**Comment by a-drew** [comment]
@mattisssa good to go!

**Pull request merged**